app/javascript/components/recipes/index/recipes-list/recipes-item.vue

```diff
@@ -1,6 +1,8 @@
 <template>
   <div class="flex flex-row divide-x-2 divide-gray-400 border-2 border-solid border-gray-400 box-border my-2">
-    <div class="flex flex-row w-10/12 justify-between items-center p-6 static h-24 bg-white flex-none flex-grow-0 self-stretch">
+    <div class="flex flex-row w-10/12 justify-between items-center p-6 static h-24 bg-white flex-none flex-grow-0 self-stretch"
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'class' should be on a new line                      vue/max-attributes-per-line